xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -1257,6 +1257,7 @@ platform.web.editors.class.pageTitle=Editing class {0}
 platform.web.editors.rights.pageTitle=Editing access rights for {0}
 platform.web.editors.unknown.pageTitle=Editing {0}
 
+core.editors.content.title=Editing title and content of <a href="{1}">{0}</a>
```

Comment:
Why providing a link there? I mean you're already on the page when you edit it...

Reply:
It's the same pattern as the one used for other edit modes:
* `core.editors.object.title=Editing objects of <a href="{1}">{0}</a>`
* `core.editors.class.title=Editing class <a href="{1}">{0}</a>`
* `core.editors.rights.title=Editing rights of <a href="{1}">{0}</a>`

The links provided redirect to the view mode. I don't think it's really necessary but we shouyld be consistent and change the others too if we don't want a link here.

Reply:
Yep agreed for consistency